packages/ui/src/components/Channel/ChannelHeader.tsx

```diff
@@ -29,31 +30,35 @@ export function ChannelHeader({
   return (
     <View paddingTop={insets.top}>
       <XStack
-        justifyContent="space-between"
         alignItems="center"
-        paddingVertical="$m"
-        paddingHorizontal="$xl"
         gap="$m"
-        borderBottomWidth={1}
-        borderBottomColor="$border"
         height="$4xl"
+        justifyContent="space-between"
+        paddingHorizontal="$xl"
+        paddingVertical="$m"
       >
         <XStack alignItems="center" gap="$m" flex={1}>
           <IconButton onPress={goBack}>
             <ChevronLeft />
           </IconButton>
-          <SizableText
-            flexShrink={1}
-            numberOfLines={1}
-            color="$primaryText"
-            size="$m"
-            fontWeight="500"
+          <Animated.View
```

Comment:
This looks nice

Reply:
Thanks. I think we're gonna explore something else but if I were to do it again, I would clip it in a rectangle and have it slide in-up/out-up without an opacity fade (similar to a slot machine). Maybe next time...